### ISSUE TITLE
Homepage - a11y improvements

### ISSUE THREAD

**Issue by mikenikles** [gitpod-io/website#38]
Feedback from @estelle 🙏 

I tested the button after our call. The button yellow (#ffb45b) with the black passes a11y color contrast. The only section that doesn’t pass is the crossed out text, but since it’s 1) crossed out, and 2) huge, it is fine. though if we want a 100 on lighthouse, that would need to be changed along with the “link with Bitbucket” and “Link with Gitlab” on the home page, and the asterisk on careers” in the footer.

https://www.youtube.com/watch?v=ZDjCO9gmlzY was a fantastic talk on color system accessibility and explained how yellow is difficult to make accessible, but the contrast is with black, so we’re all good.

**Comment by nisarhassan12** [comment]
Closing this one as we probably don't want to change the colour of the strike-through text cc @ChristinFrohne and the colour for Gitprovider buttons is reported in https://github.com/gitpod-io/website/issues/118 as well.
